gnd/src/main/java/com/google/android/gnd/model/observation/DateResponse.java

```diff
@@ -0,0 +1,86 @@
+/*
+ * Copyright 2021 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.google.android.gnd.model.observation;
+
+import androidx.annotation.NonNull;
+import com.google.android.gnd.model.form.Field;
+import java.text.SimpleDateFormat;
+import java.util.Calendar;
+import java.util.Date;
+import java.util.Locale;
+import java.util.Map;
+import java8.util.Optional;
+
+/** A user provided response to a date {@link Field}. */
+public class DateResponse implements Response {
+
+  private final Map<String,Long> epochTime;
+  public DateResponse(Map<String ,Long> time) {
```

Comment:
Why are date and time passed to the constructor as `Map` rather than `long` or some other type?